### DISCUSSION TITLE
Data filter by User Tenant

### DISCUSSION THREAD

**Discussion by Sharafudeen** [oqtane/oqtane.framework#2123] [General]
Hi,

I've created customer module, in the Index.razor component I've a datagrid with data bind from EF Core dbcontext. I want to filter data by logged in user with tenant id. How can I do it?

**Comment by sbwalker** [comment]
@Sharafudeen did you use the Create Module feature in the framework to scaffold the module code?

**Reply by Sharafudeen** [reply]
Yes I'm creating the module features from the Oqtane framework.

**Reply by sbwalker** [reply]
@Sharafudeen I am asking specifically about the Create Module option in the Module Management admin dashboard:

![image](https://user-images.githubusercontent.com/4840590/162954820-4cf75ba6-ae2b-40e6-b844-40ec0e21d605.png)

If you use this option, it will scaffold all of the code needed to build a custom module and wire up all of the dependencies to simplify development.
